lib/Bort.pm

```diff
@@ -345,14 +347,19 @@ sub load_team {
 
 
 sub my_user      { $my_user }
-sub channel_name { shift; my $channel = $_[-1]; $channel_names{$channel} // $channel }
-sub user_name    { shift; my $user = $_[-1]; $user_names{$user} // $user }
+sub channel_name { shift; my $channel = $_[-1]; $channel_names{$channel} // 0 }
```

Comment:
I think I'd prefer `()` over `0`, so we can do a definedness test on the result.

Reply:
Discussion: `()` ace